rollup.root.config.js

```diff
@@ -20,17 +21,23 @@ function init(theirConfig = {}) {
   const rollupOptionsArray = [];
   const config = Object.assign({
     addEOLocale: true, // Add the EO locale if it does not exist
-    addKELocale: true, // Add the KE locale if it does not exist
-    alwaysReturnFile: true,
-    buildTypes: [ BUILD_TYPES.MJS, BUILD_TYPES.IIFE ],
+    addKELocale: false, // Add the KE locale if it does not exist
+    alwaysReturnFile: false,
+    buildTypes: [ BUILD_TYPES.MJS, BUILD_TYPES.CJS ],
```

Comment:
`IIFE` doesn't belong in buildTypes anymore?

Reply:
This is the DEFAULT set of builds. We have always overridden them.